apps/new/widget/page/projects/Editor.jsx

```diff
@@ -36,7 +49,7 @@ const isNearAddress = (address) => {
     return false;
   }
 
-  const parts = address.split(".");
+  const parts = (address ?? "").split(".");
```

Comment:
When do we expect a null address?

Reply:
it's a fail safe, in case the address is not defined.